### PR TITLE
Add ability to load protos that depend on common protos

### PR DESCRIPTION
**Pull request by landrito**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# google-proto-files [![Build Status][build_img]][build_link]
+# nodejs-proto-files [![Build Status][build_img]][build_link]
```
Comment (stephenplusplus) [inline readme.md]: I think the name can stay. One reason would be it's the name of the module on `npm`. Another, these aren't "node" proto files, but very specific to Google and its APIs.

File: load.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+var read = require('fs-readdir-recursive')
```
Comment (stephenplusplus) [inline load.js]: Can we try to use [`globby`](https://github.com/sindresorhus/globby) instead? I believe we would do: (untested)

```js
var COMMON_PROTO_DIRS = [
  // This list of directories is defined here:
  // https://github.com/googleapis/googleapis/blob/master/gapic/packaging/common_protos.yaml
  'api',
  path.join('logging', 'type'),
  'longrunning',
  'protobuf', // This is an additional path that the common protos depend on.
  'rpc',
  'type'
].map(dir => path.join('google', dir, '**/*.proto'))

globby(COMMON_PROTO_DIRS).then(dirs => /* should already be flattened */)
```

Comment (landrito) [reply load.js]: Ohhh nice. Done!

File: readme.md (hunk 1 above)
Comment (landrito) [reply readme.md]: Fixed.

**Pull request merged**